antismash/modules/rrefinder/html_output.py

```diff
@@ -0,0 +1,33 @@
+# License: GNU Affero General Public License v3 or later
+# A copy of GNU AGPL v3 should have been included in this software package in LICENSE.txt.
+
+""" Manages HTML construction for the RREFinder module
+"""
+
+from typing import List
+
+from antismash.common import path
+from antismash.common.html_renderer import HTMLSections, FileTemplate
+from antismash.common.layers import RegionLayer, RecordLayer, OptionsLayer
+
+from .rrefinder import is_ripp, RREFinderResults
+
+
+def will_handle(products: List[str]) -> bool:
+    """ Returns True if the products provided are relevant to the module """
+    return any([is_ripp(product) for product in products])
```

Comment:
This could be a generator instead of a list comprehension, for better performance.